modules/cnv-adding-virtio-drivers-vm-yaml.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv-installing-virtio-drivers-on-existing-windows-vm.adoc
+// * cnv_users_guide/cnv-installing-virtio-drivers-on-new-windows-vm.adoc
+
+[id="cnv-adding-virtio-drivers-vm-yaml_{context}"]
+= Adding VirtIO drivers container disk to a virtual machine
+
+{ProductName} distributes VirtIO drivers for Microsoft Windows as a 
+container disk, available from the Red Hat Container Catalog. To install these 
+drivers to a Windows virtual machine, attach the `virtio-win` container disk to 
+the virtual machine as a SATA CD drive in the virtual machine configuration file. 
+
+[NOTE]
+====
+This procedure presumes the virtual machine is not running when you add the 
+container disk. If you add the container disk to a running virtual machine, you 
+can use `oc apply -f <vm.yaml>` in the CLI or reboot the virtual machine for the
+ changes to take effect.
+====
+
+.Procedure
+
+* Add the `cnv-tech-preview/virtio-win` container disk as a `cdrom` disk in the 
```

Comment:
It is part of the Red Hat container catalogue. It took me a while to scan it. The information from the first line was not clearly visible for me. Although the information is present, it should be more visible. In my opinion, Red Hat catalogue should be a requirement, which would place it more in front of the eyes.

Reply:
I've made a couple of changes:
* changed all mentions of the 'virt-win' container disk to include the 'cnv-tech-preview' namespace. Although the namespace will likely change soon, it's needed for the yaml and this keeps it consistent throughout. I think this change,in context, makes t easier to follow.
* Also added the suggested prerequisite. It's a bit of an odd one, because it's not necessary, since the container will be downloaded if it's not in the cluster, but at 150mb I guess it's big enough to speed up the procedure.
I've posted the build here: file:///home/aburden/git/openshift-docs/_preview/openshift-enterprise/cnv1610-virtio-drivers/cnv/cnv_users_guide/cnv-installing-virtio-drivers-on-existing-windows-vm.html#cnv-adding-virtio-drivers-vm-yaml_cnv-installing-virtio-drivers-on-existing-windows-vm
(There's something screwy with the css when I transfer to that fileserver. It doesn't render like that locally, nor will it once it's merged)